conjure-verification-api/Cargo.toml

```diff
@@ -4,13 +4,12 @@ version = "0.1.0"
 authors = ["Steven Fackler <sfackler@palantir.com>"]
 edition = "2018"
 publish = false
-links = "conjure-verification" # a hack to pass info upstream to conjure-verification's build script
+links = "conjure-verification"                       # a hack to pass info upstream to conjure-verification's build script
```

Comment:
Was this formatting change intentional? Are you trying to right align the comment?

Reply:
Bad default formatting setting in a TOML plugin - fixed.